charts/datafold/charts/postgres/templates/service.yaml

```diff
@@ -1,15 +1,14 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "postgres.fullname" . }}
+  name: {{ include "datafold.postgres.server" . }}
   labels:
     {{- include "postgres.labels" . | nindent 4 }}
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.service.port }}
-      targetPort: {{ .Values.service.port }}
+    - port: {{ include "datafold.postgres.port" . }}
```

Comment:
Question, could this also be 5432? So it would be a port mapping like `5432:5432` (since targetPort: postgres evaluates to 5432). Or is that not possible?

Reply:
@cfernhout : Almost. I think in kubernetes there are multiple mappings, probably more of a 5432:5432:5432 mapping. So internal port is 5432. The deployment then states it wants to open a port 5432 on its container. But that is internal to cluster only. That port on the container gets a name (probably a port name global to the application?).  Then the port in the service maps another port (clusterIP) with number 5432 to connect to 5432 on the IP of the internal docker container. 

Then through ingest you can route traffic that way.

In specific cases, not recommended, you can use nodeport. NodePort is more useful for local development, because it exposes to port to your localhost (otherwise you have to go through a more complicated ingest).